src/components/LeftSidebar/NewGroupConversation/SetContacts/SetContacts.vue

```diff
@@ -0,0 +1,126 @@
+<!--
+  - @copyright Copyright (c) 2019 Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div class="set-contacts">
+		<input
+			v-model="searchText"
+			class="set-contacts__input"
+			type="text"
+			autofocus
+			:placeholder="t('spreed', `Search Participants`)"
+			@input="handleInput">
+		<Caption
+			:title="t('spreed', `Add participants to ${conversationName}`)" />
```

Comment:
See https://github.com/nextcloud/spreed/pull/2499/files#r355394368
This will never translate

Reply:
This has been dropped after a design discussion with Jenni